### DISCUSSION TITLE
How to add annotations without “Author” tag?

### DISCUSSION THREAD

**Discussion by rgabbo** [sumatrapdfreader/sumatrapdf#2429] [Q&A] [answered]
In Windows 10, I noticed that if you add an annotation (for example highlighting a word) to a pdf file, an author tag is saved with the name of my device. If I open a pdf with annotations made using Edge built-in pdf reader there is no author tag. Is there an option in Sumatra to disable this and make annotations without it?
Sorry if this was already asked, but didn’t find anything similar in previous discussions.

**Comment by GitHubRulesOK** [comment, marked as answer]
Pick what you want from 
```
Annotations [
	HighlightColor = #ffff00
	UnderlineColor = #00ff00
	TextIconColor = #ffff00
	TextIconType = comment
	DefaultAuthor = (none)
]
```

**Reply by rgabbo** [reply]
Sorry can you elaborate? I added those strings to advanced settings but my device name is still embedded in new annotations. Thanks

**Reply by GitHubRulesOK** [reply]
You may need to double check that they have ***stuck*** when changed. So close SumatraPDF restart it again, and recheck that **(none)** has not defaulted to nothing which is current system user.

**Reply by rgabbo** [reply]
I checked that If I restart SumatraPDF, strings in advanced config are not reverted to their original state but "author" continue to be present if I create a new annotation (see pic)

![Immagine](https://user-images.githubusercontent.com/58396985/159167975-9668fec8-07f3-44c5-ba35-3d4a6d52812a.jpg)

**Reply by GitHubRulesOK** [reply]
Hmm perhaps check your using a more recent version  as ability was added [Sep 12, 2021](https://github.com/sumatrapdfreader/sumatrapdf/commit/0adbbc8308d212939eb0395d090fb09c74deb89e)

![image](https://user-images.githubusercontent.com/12754913/159168755-78620406-4664-4290-bc04-2e68da7774ea.png)

**Reply by rgabbo** [reply]
Yes, I was using latest stable. The pre-release version work as intended. Thanks for the help.
